packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -27,3 +27,14 @@ storiesOf('Components|Media Player', module).add('Audio Skin', () => (
     skin="audio"
   />
 ));
+
+storiesOf('Components|Media Player', module)
+  .add(ampDecorator)
+  .add('Audio Skin AMP', () => (
+    <AmpMediaPlayer
+      isAmp
+      src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp"
```

Comment:
```suggestion
      src="https://www.test.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio/amp"
```

Can this be changed to the liveradio player

Reply:
Done 👍